docs/HowToGuides/GettingStarted.md

```diff
@@ -305,16 +308,25 @@ It is even possible to integrate the Ninja build into Xcode. For details on how
   In many situations, there are several errors, so scrolling further back
   and looking at the first error may be more helpful than simply looking
   at the last error.
-- Check if others have encountered the same issue on the Swift forums or on
-  [Swift repository 'Issues' tab][Swift Issues].
-- Create a new Swift forums thread in the Development category. Include
-  information about your configuration and the errors you are seeing.
-  - You can [create a gist](https://gist.github.com) with the entire build
-    output and link it, while highlighting the most important part of the
-    build log in the post.
-  - Include the output of `utils/update-checkout --dump-hashes`.
+- Check if there is already a [Swift Forums] thread from someone else with the same issue.
```

Comment:
The current phrasing feels more streamline to me, and we also point to GH issues. Was there something wrong with it?

Reply:
I was thinking in guide those sort of "getting help" with build issues to the forums which feels like a more suitable place to get actual guidance on some issue. And from there after discussion, people could create GH issues if they feel is an actual build-script or actual build issue that should be fixed. But if you think is still worth it I can add it back :)

Reply:
Right, but we are only suggesting to *search* for issues besides forum threads, not create them.